nx.json

```diff
@@ -212,7 +225,13 @@
       "!{projectRoot}/.storybook/**/*",
       "!{projectRoot}/tsconfig.storybook.json",
       "!{projectRoot}/src/test/**/*"
-    ]
+    ],
+    "defaultPython": [
+      "{projectRoot}/**/*",
+      "{workspaceRoot}/pyproject.toml",
+      "{workspaceRoot}/uv.lock"
+    ],
+    "productionPython": ["defaultPython"]
```

Comment:
We can refine this value in the future to exclude test files and others using the "production" value designed for TypeScript projects.